test/Browser/DICOMSeriesTest.js

```diff
@@ -24,7 +24,7 @@ test('Test reading DICOM file series', t => {
     })
     .then(function ({ image, webWorkerPool }) {
       t.is(image.imageType.dimension, 3, 'dimension')
-      t.is(image.imageType.componentType, IntTypes.UInt8, 'componentType')
+      t.is(image.imageType.componentType, IntTypes.Int16, 'componentType')
```

Comment:
Why would those change? The pixel type should be defined by [PixelRepresentation](https://dicom.innolitics.com/ciods/ct-image/image-pixel/00280103) and [BitsAllocated](https://dicom.innolitics.com/ciods/ct-image/image-pixel/00280100).

Reply:
Likely related to how rescale slope / intercept is applied.